### DISCUSSION TITLE
I can't get any other ideas on target #30!!

### DISCUSSION THREAD

**Discussion by joewinbpaul** [cssbattle/forum#331] [Q&A] [answered]
I can't get better solutions than the 89-char solution:
![30](https://user-images.githubusercontent.com/102139966/206834344-d2f224b0-805b-4de7-93be-1967b515996f.png)

"<style>*+*{background:linear-gradient(#F7EC7D 53q,#E38F66 0 106q,#AA445F 0 67%,#62306D 0"    
How on earth did they get less?!?!     
Are they even using gradients btw?

**Comment by verberg** [comment]
I'm just wondering too!

**Comment by Gabb192** [comment, marked as answer]
@joewinbpaul You are pretty close to the best solution! Unfortunately now we can only reach 84 chars, 82 is no more available due to an rgba that is not working anymore. The best approach here is `box-shadow` with some nice unit replacement.

**Reply by joewinbpaul** [reply]
> @joewinbpaul You are pretty close to the best solution! Unfortunately now we can only reach 84 chars, 82 is no more available due to an rgba that is not working anymore. The best approach here is `box-shadow` with some nice unit replacement.

THX I am now stuck with 88 chars using `box-shadow` 😶  Any other clues ? 😅😅

**Reply by joewinbpaul** [reply]
> > @joewinbpaul You are pretty close to the best solution! Unfortunately now we can only reach 84 chars, 82 is no more available due to an rgba that is not working anymore. The best approach here is `box-shadow` with some nice unit replacement.
> 
> THX I am now stuck with 88 chars using `box-shadow` 😶 Any other clues ? 😅😅

I think u mean we should convert it to a 4-digit hex code but I don't know how to do that <p> Is there some sort of a calculator for it somewhere?

**Reply by Gabb192** [reply]
@joewinbpaul Now you just have to find the perfect margin value (with the best selector) so you can replace 3 shadows with cm/in instead of q.
Regarding 4-digit hex color you could use this web site from @SelenIT `https://codepen.io/SelenIT/pen/JaeYpM?editors=0012` but as I said there is no working RGBA here.

Also I suggest you to join the Discord server, you will find many answer there!

**Reply by joewinbpaul** [reply]
> @joewinbpaul Now you just have to find the perfect margin value (with the best selector) so you can replace 3 shadows with cm/in instead of q. Regarding 4-digit hex color you could use this web site from @SelenIT `https://codepen.io/SelenIT/pen/JaeYpM?editors=0012` but as I said there is no working RGBA here.
> 
> Also I suggest you to join the Discord server, you will find many answer there!

Thanks @Gabb192
